src/Integration.Vsix/Resources/Strings.resx

```diff
@@ -151,6 +155,14 @@
   <data name="Daemon_Installing" xml:space="preserve">
     <value>Installing the daemon...</value>
   </data>
+  <data name="Daemon_InvalidFileNameInDownloadEnvVar" xml:space="preserve">
+    <value>The daemon file name supplied in {0} environment variable is invalid ('{1}'). Using the default download URL.</value>
+    <comment>Output window message</comment>
+  </data>
+  <data name="Daemon_InvalidUrlInDownloadEnvVar" xml:space="preserve">
+    <value>Download URL supplied in {0} environment variable is invalid ('{1}'). Using default download URL.</value>
```

Comment:
is it worth logging the default download URL as well? (applies for above message, too)

Reply:
The actual version and url that end up being used are logged by the constructor, so that information will be in the output.